gormschema/gorm.go

```diff
@@ -38,6 +40,13 @@ func WithConfig(cfg *gorm.Config) Option {
 	}
 }
 
+// WithCreateConstraintsAfterCreateTable sets the createConstraintsAfterCreateTable config.
+func WithCreateConstraintsAfterCreateTable(b bool) Option {
+	return func(l *Loader) {
+		l.createConstraintsAfterCreateTable = b
+	}
+}
```

Comment:
ditto. It does not look like Atlas API. 


Also, why the user should be aware of this? This is encapsulated by Atlas and should be different here

Reply:
I'll remove it